lib/RuntimeBabel.js

```diff
@@ -259,8 +259,11 @@ module.exports = function(S) {
      * - removes all non-bundled .js and .json files from the pathDist
      */
 
-    _cleanup(pathDist) {
-      let ignore = '**/_serverless_handler.js';
+    _cleanup(func, pathDist) {
+      let exclude = _.get(func, 'custom.runtime').exclude || [],
+        ignore = ['**/_serverless_handler.js']
+          .concat(exclude.map(ex => `${pathDist}/**/node_modules/${ex}/**/*`));
```

Comment:
Maybe the regex is not perfect.

my exclude is 
["regenerator-runtime/path.js", "regenerator-runtime/runtime.js"]
(i tried to make pac-resolver work, because of the readFileSync)

my structure is
project root
---server
------node_modules (just for the api)
------api

and i add to add
.concat(exclude.map(ex => `${pathDist}/node_modules/${ex}`));